tests/api_tests/collections/test_views.py

```diff
@@ -0,0 +1,622 @@
+        assert_not_in(self.private_collection_user_two._id, ids)
+        assert_not_in(self.folder._id, ids)
+        assert_not_in(self.dashboard._id, ids)
+
+    def test_get_one_collection_with_exact_filter_logged_in(self):
+        url = "/{}collections/?filter[title]=Project%20One".format(API_BASE)
+
+        res = self.app.get(url, auth=self.basic_auth_one)
+        node_json = res.json['data']
+
+        ids = [node['id'] for node in node_json]
+        assert_in(self.collection_one._id, ids)
+        assert_not_in(self.collection_two._id, ids)
+        assert_not_in(self.collection_three._id, ids)
+        assert_not_in(self.private_collection_user_one._id, ids)
+        assert_not_in(self.private_collection_user_two._id, ids)
+        assert_not_in(self.folder._id, ids)
+        assert_not_in(self.dashboard._id, ids)
+
+    def test_get_some_collections_with_substring_logged_in(self):
+        url = "/{}collections/?filter[title]=Two".format(API_BASE)
+
+        res = self.app.get(url, auth=self.basic_auth_one)
+        node_json = res.json['data']
+
+        ids = [node['id'] for node in node_json]
+        assert_not_in(self.collection_one._id, ids)
+        assert_in(self.collection_two._id, ids)
+        assert_not_in(self.collection_three._id, ids)
+        assert_not_in(self.private_collection_user_one._id, ids)
+        assert_not_in(self.private_collection_user_two._id, ids)
+        assert_not_in(self.folder._id, ids)
+        assert_not_in(self.dashboard._id, ids)
+
+    def test_get_only_public_or_my_collections_with_filter_logged_in(self):
+        url = "/{}collections/?filter[title]=Project".format(API_BASE)
+
+        res = self.app.get(url, auth=self.basic_auth_one)
+        node_json = res.json['data']
+
+        ids = [node['id'] for node in node_json]
+        assert_in(self.collection_one._id, ids)
+        assert_in(self.collection_two._id, ids)
+        assert_not_in(self.collection_three._id, ids)
+        assert_in(self.private_collection_user_one._id, ids)
+        assert_not_in(self.private_collection_user_two._id, ids)
+        assert_not_in(self.folder._id, ids)
+        assert_not_in(self.dashboard._id, ids)
+
+    def test_incorrect_filtering_field_logged_in(self):
+        # TODO Change to check for error when the functionality changes. Currently acts as though it doesn't exist
+        url = "/{}collections/?filter[notafield]=bogus".format(API_BASE)
+
+        res = self.app.get(url, auth=self.basic_auth_one)
+        node_json = res.json['data']
+
+        ids = [node['id'] for node in node_json]
+        assert_in(self.collection_one._id, ids)
+        assert_in(self.collection_two._id, ids)
+        assert_in(self.collection_three._id, ids)
+        assert_in(self.private_collection_user_one._id, ids)
+        assert_not_in(self.private_collection_user_two._id, ids)
+        assert_not_in(self.folder._id, ids)
+        assert_not_in(self.dashboard._id, ids)
+
```

Comment:
We discussed that a user must be logged in to get the list and the documentation was changed to reflect this. Is this no longer the case?